simmechanics_to_urdf/firstgen.py

```diff
@@ -1839,15 +1856,19 @@ class URDFSensorsGenerator:
     def __init__(self):
         self.dummy = ""
 
-    def getURDFForceTorque(self, jointName, sensorName, directionChildToParent):
+    def getURDFForceTorque(self, jointName, sensorName, directionChildToParent, frame):
```

Comment:
We need to pass the sensor pose also here, otherwise the sensor will not work. 

See the `origin_el`  in the `getURDFSensor` method of this class to understand how to do it.